README.md

```diff
@@ -136,3 +136,9 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ```
+
+Maintainers
+-----------
+* Sean O'Meara
+* Steve Murawski
+* Jerrel Blankenship (<jerrel@jerrelblankenship.com>)
```

Comment:
hey @jerrelblankenship - would you prefer a link to your github profile over an embedded email address?   let's add that for @someara and @smurawski as well.